packages/nitro-protocol/test/contracts/ForceMove/conclude.test.ts

```diff
@@ -45,8 +44,7 @@ for (let i = 0; i < 3; i++) {
 beforeAll(async () => {
   networkMap = await getNetworkMap();
   ForceMove = await setupContracts(provider, ForceMoveArtifact);
-  networkId = (await provider.getNetwork()).chainId;
-  appDefinition = networkMap[networkId][countingAppArtifact.contractName]; // use a fixed appDefinition in all tests
+  appDefinition = networkMap[countingAppArtifact.contractName].address; // use a fixed appDefinition in all tests
```

Comment:
Maybe you can get rid of the duplicated comments by importing the counting app as `placeholderArtifact` or something?

Reply:
Done.